test/acceptance/keywords/javascript.robot

```diff
@@ -107,3 +107,12 @@ Drag and Drop by Offset
     Element Text Should Be    id=droppable    Drop here
     Drag and Drop by Offset    id=draggable    ${100}    ${20}
     Element Text Should Be    id=droppable    Dropped!
+
+Verify Console Log Can be Caught
+   ${message}  Set Variable   Sample Console Error
+   Execute Javascript  console.error('${message}')
+   ${logs}=  Get Log  browser
```

Comment:
We should have tests for all possible cases. In this test `browser` would not contain the expected message, "Sample Console Error". (It did show in Firefox console).
Did some trials with `driver` and it got output, but with `client` and `server` I've got zero size ${logs}.

The test is failing at CI, because the log entry you are getting is a localized message about no specification of the file encoding (and not containing the expected message). In my system and in PT, is:
`'A codificação de caracteres do documento HTML não foi declarada. O documento será mostrado com texto ilegível em algumas configurações de navegadores se o documento tiver caracteres fora do intervalo US-ASCII. A codificação de caracteres da página tem de ser declarada no documento ou no protocolo de transferência.' `